src/app/core/http/http.service.mock.ts

```diff
@@ -0,0 +1,202 @@
+import { Injectable } from '@angular/core';
+import { Event } from '../../shared/models/event';
+import { Observable, of } from 'rxjs';
+import { Email, PasswordReset, User } from '../../shared/models/user';
+import { EventParticipant, EventParticipantAnswer } from '../../shared/models/event-participant';
+import { MailInvite } from '../../shared/models/mailInvite';
+import { DistributionList, DistributionListRequest } from '../../shared/models/distribution-list';
+
+@Injectable()
+export class HttpService {
+
+  private users: User[] = [
+    {
+      ref: "user1",
+      email: "email@no.domain.com",
+      firstName: "First",
```

Comment:
can we put something that looks like a real name as it will be displayed in the profile page, something like 'John Smith'